frontend/src/pages/CourseDashboard/constants.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/prefer-default-export */
+import {
```

Comment:
Is this going to be exactly the same as the resource dashboard? Can we just use that config over?

Reply:
Wasn't sure of the direction of this page so I created its own config.